resources/views/pages/index.blade.php

```diff
@@ -4,80 +4,86 @@
 @section('description', $page['meta_description'])
 @section('keywords', $page['meta_keywords'])
 @if(!empty($page['hero_image']))
-    @section('hero_image', $page['hero_image']['data']['url'])
-    @section('hero_image_title', $page['hero_image_alt_text'])
+@section('hero_image', $page['hero_image']['data']['url'])
+@section('hero_image_title', $page['hero_image_alt_text'])
 @else
-    @section('hero_image','https://fitz-cms-images.s3.eu-west-2.amazonaws.com/img_20190105_153947.jpg')
-    @section('hero_image_title', "The inside of our Founder's entrance")
+@section('hero_image','https://fitz-cms-images.s3.eu-west-2.amazonaws.com/img_20190105_153947.jpg')
+@section('hero_image_title', "The inside of our Founder's entrance")
 @endif
 
 @section('content')
-    @if(!empty($page['carousel_associated']))
-        @if($page['carousel_associated'][0]['carousels_id'])
-            <div class="container-fluid">
-                <div class="negative-padding">
-                    @include('includes.structure.carousel-pages')
-                </div>
-            </div>
-        @endif
+@if(!empty($page['carousel_associated']))
+@if($page['carousel_associated'][0]['carousels_id'])
```

Comment:
I would recommend indenting these if statements to look similar to how they were before the changes were made, it helps a lot with readability.

e.g. at the moment it is like this:

```
@section('content')
@if(!empty($page['carousel_associated']))
@if($page['carousel_associated'][0]['carousels_id'])
@endif 
@endif 
@endsection 
```

I would recommend changing it to like this:

```
@section('content')
    @if(!empty($page['carousel_associated']))
        @if($page['carousel_associated'][0]['carousels_id'])
        @endif 
    @endif 
@endsection 
```